app/scripts/configs/tracks-info.js

```diff
@@ -223,10 +221,12 @@ export const TRACKS_INFO = [
     }
   },
   {
-    type: 'horizontal-heatmap',
+    type: 'linear-heatmap',
+    aliases: ['horizontal-heatmap', 'vertical-heatmap'],
     datatype: ['matrix'],
     local: false,
     orientation: '1d-horizontal',
```

Comment:
this is only kept for backward compatibility right? If so, how about adding a comment so we know when we look at the file in the future.

```suggestion
    orientation: '1d-horizontal',  // only for backward compatibility
```

Even though it'd be verbose but I'd add the same comment everywhere it matches.

I just bring this up because we have so many props and I kind of lost track which one is actually still used and which one is only there for older version of glib. (e.g., I never know what `local: false` is used for; not saying we need to fix that right now)

Reply:
Done 👍